Sources/PodToBUILD/ShellContext.swift

```diff
@@ -46,7 +46,7 @@ let FOUR_HOURS_TIME_CONSTANT = (60.0 * 60.0 * 4.0)
 public protocol ShellContext {
     func command(_ launchPath: String, arguments: [String]) -> CommandOutput
 
-    func shellOut(_ script: String) -> CommandOutput
+    @discardableResult func shellOut(_ script: String) -> CommandOutput
```

Comment:
Command should also probably have `@discardableResult`, will add this before merging.